src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        boolean isHeader = true; // Skip the header line
+        for (String line : data) {
+            if (isHeader) {
+                isHeader = false;
+                continue;
+            }
```

Comment:
Using a boolean flag 'isHeader' to skip the header is not the most elegant solution. Consider using a more expressive way, like removing the header from the list before processing or starting the loop from the second element if the header is always expected.